src/module.c

```diff
@@ -1008,136 +1036,150 @@ int RediSearch_InitModuleInternal(RedisModuleCtx *ctx, RedisModuleString **argv,
 #define INDEX_DOC_CMD_ARGS 2, 2, 1
 #endif
 
-  RM_TRY(RedisModule_CreateCommand, ctx, RS_INDEX_LIST_CMD, IndexList, "readonly", 0, 0, 0);
+  // Create the `search` ACL command category
+  if (RedisModule_AddACLCategory(ctx, "search") == REDISMODULE_ERR) {
+      RedisModule_Log(ctx, "warning", "Could not add `search` ACL category, errno: %d\n", errno);
```

Comment:
Should not be a warning if this will prevent the module from being loaded and can also prevent a server from starting,

Reply:
Why not?

Reply:
I thought it should be an error but I see the vast majority of what we currently use is "warning"

Reply:
Yes, anyways the server will fail to load.